### ISSUE TITLE
CodeMirror hint box should be wider

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#17409]
### Describe the bug

CodeMirror hint box should be wider in some cases, because column names are overlapping with column types.

### To Reproduce

1. Go to [https://demo.phpmyadmin.net/master-config/index.php?route=/table/sql&db=sakila&table=staff](https://demo.phpmyadmin.net/master-config/index.php?route=/table/sql&db=sakila&table=staff)
2. Write `` to open the hint select
3. See the column name overlapping with column type

### Expected behavior

The hint box should be wider.

### Screenshots

https://user-images.githubusercontent.com/25424343/154828083-6e77cce9-241b-44c7-b572-72184f834dcd.mp4

### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version:

### Client configuration

- Browser:
- Operating system:

### Additional context

Add any other context about the bug here.

**Comment by ValkonX33** [comment]
hey can you assign me this?

**Comment by liviuconcioiu** [comment]
I've updated the initial report. It happens on Firefox.

**Comment by liviuconcioiu** [comment]
@ValkonX33 you have to fork phpMyAdmin. You should create a new branch from QA_5_2 branch (this is were the current issue should be solved) and then open a new PR with your branch name, by targeting QA_5_2.

Regarding the issue, if it's a theme one, you should check all the available themes, to see if everything is okay.

**Comment by alexlaslau** [comment]
Hello! Does this issue still require solving?
This would be my first time contributing to open source and I'm not sure exactly if you have to ask before starting to work on an issue 😇

**Comment by liviuconcioiu** [comment]
Hi @alexlaslau! Feel free to submit a PR. There are no assignments on this project and everyone is free to submit a PR.

If an issue is present in 5.2.2-dev you should target QA_5_2 branch. If is on 6.0.0-dev you should target master branch.

**Comment by hamza-hafeez1** [comment]
Hi @liviuconcioiu , Is this issue still open? I am new to contributing and would like to work on this issue as my first one.

**Comment by liviuconcioiu** [comment]
Yes
